### ISSUE TITLE
Method documentation slightly messed up with union types

### ISSUE THREAD

**Issue by flack** [eclipse/pdt#166]
**Describe the bug**

If you have a function like `array_key_exists`, where the `$key` parameter can be `int|string`, then this is not correctly shown in the tooltips, and autocompletion ignores the union type parameter. 

**Describe the eclipse environment**
Eclipse 23.03
Eclipse PDT 8.0.0.202305212131

**Describe your system**
 - OS: Ubuntu
 - Version 23.04

**To Reproduce**
For example, if you type

```
array_k
```

then wait for the list of suggestions to come up, hit Enter to select `array_key_exists`, it will produce this:

```
array_key_exists($array)
```

instead of the expected

```
array_key_exists($key, $array)
```

**Expected behavior**
Union type parameters should work like the others

**Screenshots**

![Screenshot_20230523_150238](https://github.com/eclipse/pdt/assets/425166/03f493d2-daf8-4fbc-8727-98f25192e1e0)

**Comment by zulus** [comment]
Good catch! But the problem is in different place. PHP indexer use PIPE as parameter separator xD
